### PR TITLE
add Google Tag Manager

### PR DESCRIPTION
**Pull request by rosejbon**
## Description

- Added tag manager and removed GA script as I have now connect tag manager to Google Universal Analytics via tag manager
-

@neontribe/ca-seap

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -25,16 +25,20 @@
...
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-KB8DVFH');
     </script>
+    <!-- End Google Tag Manager -->
   </head>
   <body>
+    <!-- Google Tag Manager (noscript) -->
```
Comment (katjam) [inline src/index.html:39]: The elm will take over the body if js is running. If not, we have no app... but could be useful to track if people to hit this. Is it a unique event?

File: src/index.html [hunk 2]
```diff
@@ -25,16 +25,20 @@
...
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157133795-1"></script>
+    <!-- Google Tag Manager -->
     <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
-        gtag('js', new Date());
-
-        gtag('config', 'UA-157133795-1');
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```
Comment (katjam) [inline src/index.html:30]: The subscriptions to app events rely on `gtag` in the `index.js`

If this is all working as expected, then I guess is OK.

**Review by katjam — COMMENTED** [review]
Probably fine if it's giving the behaviour you expect.

I don't know about this update compared to the old way. Leaving unapproved until you verify that you are happy with it... and that we don't need changes to the `index.js` or core code.

File: src/index.html (hunk 1 above)
Comment (rosejbon) [reply src/index.html:39]: It still seems to work even though elm is overriding this. I don't think its an event, I was just told to put it in the body by Google, is there a place I could put it so its first in the body?

File: src/index.html (hunk 2 above)
Comment (rosejbon) [reply src/index.html:30]: So for the project I am setting up new events to track based on what I need to measure and I think these are new to the other events we set up and for seAp I don't think we need the old ones. I should show you my master analytics plan at some point :)

File: src/index.html (hunk 1 above)
Comment (rosejbon) [reply src/index.html:39]: I am not going worry about tracking this for now, will worry for Samaritans and chuck an issue in the backlog

**Comment by rosejbon** [comment]
Behaviour as excepted, Rob and I have tested, so I am fairly confident it is working :)

**Review by katjam — APPROVED** [review]
Go for it!

**Pull request merged**